internal/provider/protocols_cifs_share_resource_test.go

```diff
@@ -50,9 +62,12 @@ func TestAccProtocolsCIFSShareResource(t *testing.T) {
 }
 
 func testAccProtocolsCIFSShareResourceConfig(svm, shareName string) string {
-	host := os.Getenv("TF_ACC_NETAPP_HOST_CIFS")
-	admin := os.Getenv("TF_ACC_NETAPP_USER")
-	password := os.Getenv("TF_ACC_NETAPP_PASS2")
+	// host := os.Getenv("TF_ACC_NETAPP_HOST_CIFS")
+	// admin := os.Getenv("TF_ACC_NETAPP_USER")
+	// password := os.Getenv("TF_ACC_NETAPP_PASS2")
+	host := "10.193.73.2"
+	admin := "admin"
+	password := "Netapp1!"
```

Comment:
you need to remove the Host, username, and password and use one of the TF variable. If we need new one let me know.